### ISSUE TITLE
URL /apex/wwv_flow.ajax was not found on this server

### ISSUE THREAD

**Issue by devoracle** [Dani3lSun/apex-plugin-dropzone#62]
### Expected behavior
File should upload

### Actual behavior
Error ( please see the attachment). In browser console I could see URL /apex/wwv_flow.ajax was not found on this server error

### Steps to reproduce the issue

I have install the dropzone2 plugin (2.4.1) and created a dropzone region. Dragged and dropped the file.

### APEX version (4.2.6 / 5.0.3)
 APEX version 5.1.2.00.09

### Used web server / version and platform (ORDS 3.0.3 / Tomcat 7 / Apache 2.4 / Linux x64)

PLSQL_GATEWAY | WebDb

GATEWAY_IVERSION | 2
SERVER_SOFTWARE | Oracle Embedded PL/SQL Gateway/11.2.0.3.0
GATEWAY_INTERFACE | CGI/1.1
SERVER_PORT | 8080
SERVER_NAME | XDB HTTP Server
REQUEST_METHOD | GET

### Used web browser / version and platform (Chrome 48 Mac / Firefox 44 Windows)

Chrome 64 bit / 72.0.3626.119

![image](https://user-images.githubusercontent.com/48345258/54011863-0e2b5280-419a-11e9-938e-9fb1410638b6.png)

Could you please help me. I'm new to APEX.

Thank you in advance,
Dev

**Comment by devoracle** [comment]
I forgot mention. I'm using Upload mechanism as Normal.

**Comment by rockingeric** [comment]
I've just had the same issue and I found out that It did not work because the latest version of Apex Plugin Dropzone compatible with my APEX Version (5.0) was 2.2.4. Click here https://github.com/Dani3lSun/apex-plugin-dropzone/releases/tag/v.2.2.4 and download the Javascript for that version, located in server/js.

TL;DR: Download the 2.2.4 version of the apexdropzone.js file.

**Comment by Dani3lSun** [comment]
Hi @devoracle 
I can imagine that the "Normal" upload method has problems with your env (you are using EPG, this is a very limited embedded http server of DB). Either try using with ORDS or try to use "Chunked" method...
